src/components/space/spaceSidebarTabs/SpaceSidebarTabs.tsx

```diff
@@ -1,3 +1,88 @@
+// React + Web3 Essentials
```

Comment:
Lets make this component reusable such that we can pass in any number of tab option and map through it